### DISCUSSION TITLE
How do you enable nvidia-smi monitoring on the latest versions of netdata?

### DISCUSSION THREAD

**Discussion by TitanLabs-ai** [netdata/netdata#17467] [Q&A]
I can not figure out how to enable the nvidia-smi monitoring with the latest version?

I found this: https://learn.netdata.cloud/docs/collecting-metrics/hardware-devices-and-sensors/nvidia-gpu

It says you need to manually enable it but doesn't tell you what to actually edit/add to enable the file once you have it open?

It already has this in it:

jobs:
  - name: nvidia_smi
    use_csv_format: yes

Anyone know how to enable it?

Thanks

**Comment by ilyam8** [comment]
Hey, @TitanLabs-ai. You need to enable `nvidia_smi` in `go.d.conf`, see [Setup->Prerequisites](https://learn.netdata.cloud/docs/collecting-metrics/hardware-devices-and-sensors/nvidia-gpu#prerequisites).

And I suggest you set `use_csv_format: no`. There will be more metrics when using XML.

**Reply by TitanLabs-ai** [reply]
Thank you, I feel silly now lol.
